packages/nitro-protocol/test/src/transactions.test.ts

```diff
@@ -147,7 +148,7 @@ describe('transaction-generators', () => {
 
     it('throws an error when there is no challenge state', async () => {
       expect(() => {
-        createRespondTransaction(null, signedStateA);
+        createRespondTransaction((null as unknown) as State, signedStateA);
```

Comment:
nit but this seems like a strange cast?

Reply:
Yeah I don't think we need it in the end, but probably because we don't have a very strict typescript configuration. I think if we had stricter config, it may be necessary to avoid `tsc` complaining that `null` is "too different" to `State`. 

See https://stackoverflow.com/questions/58923573/error-ts2352-conversion-of-type-session-null-to-type-x-string-y-strin

Anyway I removed this in 5575648